hub/core/chunk_engine/util.py

```diff
@@ -0,0 +1,75 @@
+import os
+import numpy as np
+from hub import constants
+
+from typing import Tuple
+
+
+def array_to_bytes(array: np.ndarray) -> bytes:
+    return array.tobytes()
+
+
+def get_random_array(shape: Tuple[int], dtype: str) -> np.ndarray:
+    dtype = dtype.lower()
+
+    if "int" in dtype:
```

Comment:
There is no way this function can't be implemented in fewer than 5 lines

Reply:
it's actually really weird i tried finding a function that does this but there isn't one